### PR TITLE
Dept 71 search query

### PR DESCRIPTION
**Pull request by agonxalez**

### CODE REVIEW

File: components/search/index.tsx [hunk 1]
```diff
@@ -11,6 +11,8 @@ const SearchBox = () => {
   const [inputValue, setInputValue] = useState('');
   const inputRef = useRef<HTMLInputElement>(null);
   const router = useRouter();
+  const currentPage = router.pathname;
+  console.log(currentPage);
```
Comment (bdurel) [inline components/search/index.tsx:15]: remove console.log

**Review by bdurel — CHANGES_REQUESTED** [review]
Apart from removing the console.log, the code looks good. Functionally I have a couple of questions, but I'm not clear on whether these issues have to do with your card. (1) I am wondering about something that's happening when the search results load. Briefly the "Your search does not match any profiles." error flashes before search results are shown. This happens both when making the initial search and when navigating back from a user profile. I believe fixing that might be outside the scope of your card though. (2) This one might be more within the scope, since search results not appearing is mentioned in Jira. When I make a search for "a" from the search results page, then click on a see shoutouts for a user, and then click the back I get no search results like this...
image

**Comment by bdurel** [comment]
Good work on fixing that back/disappearing search results bugginess. It all looks good apart from the flashing error message. Is that going to be on a separate bugfix card? If so, this passes for me.

**Comment by bdurel** [comment]
Flashing is a separate issue from this card, so this passes.

**Review by bdurel — APPROVED** [review]

**Review by lsanchezcode — APPROVED** [review]
LGTM :)

**Pull request merged**